### PR TITLE
codespell: workflow to prevent typos, fix some (?)

### PR DESCRIPTION
**Pull request by yarikoptic**

### CODE REVIEW

File: prisma/seed.ts [hunk 1]
```diff
@@ -214,7 +214,7 @@ async function main() {
                     ],
                   },
             output: {
-              complation: `Creating a React component can be done in two ways: as a functional component or as a class component. Let's start with a basic example of both.
+              compilation: `Creating a React component can be done in two ways: as a functional component or as a class component. Let's start with a basic example of both.
```
Comment (yarikoptic) [inline prisma/seed.ts:217]: or was "completion" intended? not yet familiar with this project at all

Comment (yarikoptic) [reply prisma/seed.ts:217]: yeah, completion -- will fix up

**Comment by marcklingen** [comment]
Thanks for the contribution, let me know when I can check it out

**Comment by yarikoptic** [comment]
ok -- please check it out, I fixed it up right away

**Comment by marcklingen** [comment]
thx @yarikoptic, merged

**Comment by marcklingen** [comment]
Join us on Discord if you want to discuss further ideas on how to improve the project: https://discord.gg/7NXusRtqYU

**Pull request merged**